lib/moon/packages/data_bags/data_matrix.rb

```diff
@@ -152,7 +142,7 @@ def [](x, y, z)
       y = y.to_i
       z = z.to_i
       return @default unless contains?(x, y, z)
-      @data[calc_index(x, y, z)]
+      @data[calc_index(x, y, z)] || @default
```

Comment:
So the `contains?` check will fail?... that should at least have a comment, but it doesn't seem like a good design